sections/sprint-1/architecture.tex

```diff
@@ -10,6 +10,7 @@ \section{Architecture of the game}\label{sec:sprint1-architecture}
 In \autoref{fig:architecture} we see an illustration of the initial idea of how the different components of the game are going to work together.
 The arrows in the diagram show the flow of data in the game.
 The Pozyx component is responsible for providing the positional data to the host.
+The Pozyx component will be described into further depth in \autoref{sec:sprint1-pozyx}.
```

Comment:
```suggestion
The Pozyx component is responsible for providing the positional data to the host, this will be described in further depth in \autoref{sec:sprint1-pozyx}.
```